README.md

```diff
@@ -59,17 +67,21 @@ import '~network-canvas-ui/styles/all.scss';
 
 - LIST TBC
 
-## Contributing
+## Contributing / Development
 
 ### Components
 
-- Add your components in `/src/`, link to them in the relevant `index.js`.
+- Run `npm run dev` to run a dev server and make chnages to components and styles in `/src/`, then link to them in the relevant `index.js`.
 - Run the babel build so that they can be imported
 
   ```sh
   $ yarn run prepublish
   ````
+  or
+  ```sh
+  $ npm run prepublish
+  ````
 
 ### Styles
 
-Just add the styles in `/styles/` and link to them in the relevant `_all.scss`, and you're done!
+Just add the styles in `/styles/` and link to them in the relevant `main.scss`, and you're done!
```

Comment:
Not if we want to be able to use mixins/variables in the main app where these are imported?

It seems like maybe there are two ideas about how this would work?:
a) We import the styles into stylesheets either wholesale or on a per component basis, kind of like a library.
b) We <link> the styles into the main app (post compilation?), kind of like bootstrap.